Khan/wonder-blocks
# Fix banner stories options

## Summary:
There was an error and the dropdown was glitching whenever
selecting something from the `actions` or `onDismiss` dropdown
in the props panel for Banner.

This is fixed by adding `Object.keys` for the `options` field
and adding a `mapping` field.

Issue: https://khanacademy.atlassian.net/browse/WB-1455

## Test plan:
- Go to http://localhost:6061/iframe.html?args=kind:success;layout:floating&id=banner--default&viewMode=story
- Open the console
- Confirm there are no errors
- Go to http://localhost:6061/?path=/story/banner--default
- Open the console
- Change the dropdown values for `actions` and `onDismiss`
- Confirm that does no cause any errors
